concrete/src/Application/UserInterface/Menu/Item/Item.php

```diff
@@ -17,12 +17,12 @@ public function __construct($handle, $pkgHandle = false)
 
     public function getHandle()
     {
-        return $this->handle;
+        return isset($this->handle) ? $this->handle : null;
```

Comment:
This should be declared, not isset checked. Objects with undeclared properties end up taking waaaaaaaaay more memory to keep around: 
https://3v4l.org/Ac3ZU